### PR TITLE
CPU and Memory Metrics

### PR DESCRIPTION
**Pull request by yakovdyadkin**
- Added CPU and Memory metric collection to node exporter
- Added new Grafana dashboard for detailed CPU, Memory, and Network metrics
- Updated Nvidia Cluster View dashboard with CPU and Memory metrics
- Added Nvidia exporter job id update bugfix when profiling, @rafsalas19 
- Miscellaneous formatting changes

### CODE REVIEW

File: src/worker/exporters/base_exporter.py [hunk 1]
```diff
@@ -29,9 +29,9 @@ def init_connection(self):
 
     def init_gauges(self):
         '''Initialization of Prometheus parameters. Override in Child Class if needed'''      
-        self.guages = {}
+        self.gauges = {}
```
Comment (rafsalas19) [inline src/worker/exporters/base_exporter.py:32]: I don't see any changes in this file? what was changed? If nothing was can you revert this file back to the original.

File: src/worker/exporters/node_exporter.py [hunk 2]
```diff
@@ -106,6 +210,19 @@ def init_config(job_id, port=None):
...
 
+        elif 'cpu' in field_name:
+            if 'util' in field_name:
+                config['fieldFiles'][field_name] = '/proc/stat'
+
+                # Call cpu_percent to get intial 0.0 values
+                _ = psutil.cpu_percent(percpu=True)
+
+            elif 'frequency' in field_name:
+                config['fieldFiles'][field_name] = '/proc/cpuinfo'
```
Comment (rafsalas19) [inline src/worker/exporters/node_exporter.py:221]: Are  '/proc/cpuinfo' and '/proc/meminfo' being used? if not lets remove them from here.

File: src/worker/exporters/base_exporter.py (hunk 1 above)
Comment (yakovdyadkin) [reply src/worker/exporters/base_exporter.py:32]: Gauges was misspelled, which caused problems in the child class NodeExporter

File: src/worker/exporters/node_exporter.py (hunk 2 above)
Comment (yakovdyadkin) [reply src/worker/exporters/node_exporter.py]: Addressed

**Review by rafsalas19 — APPROVED** [review]

**Pull request merged**